examples/sugarscape_g1mt/sugarscape_g1mt/model.py

```diff
@@ -187,6 +187,13 @@ def step(self):
 
         # collect model level data
         self.datacollector.collect(self)
+        # Need to remove excess data
+        # Create local variable to store trade data
```

Comment:
What about https://github.com/projectmesa/mesa/issues/1419#issuecomment-1257158578 ? Which doesn't expose the internal wiring of the `DataCollector`.

Reply:
My concern with putting it in the model collector is the point of the lesson is to talk about the agent reporter. Showing the internal wiring of `datacollector`, I think is more consistent with the learning objectives as I show the source code in various lessons.. This leads back to your second comment that we would need to build a class for collecting `DataCollectorRABT`. 

So in I would prefer to show the internal wiring... I don't know.. what do you think?

Reply:
I suppose it is fine for now as long as there is a comment that says this part is still a work in progress, with a link to #1419.

Reply:
Comment added